idea/common/src/interfaces/meta.ts

```diff
@@ -1,7 +1,19 @@
+type HumanTypesRepr = {
+  input?: number;
+  output?: number;
+};
+
+export interface IMetaData {
+  init: HumanTypesRepr;
+  handle: HumanTypesRepr;
+  reply: HumanTypesRepr;
+  others: HumanTypesRepr;
+  signal: number | null;
+  state: number | null;
+}
+
```

Comment:
You can import these interfaces from @gear-js/api

Reply:
Done